app/transformations/LoansTransformer.scala

```diff
@@ -493,6 +493,10 @@ class LoansTransformer @Inject() extends Transformer {
         index => LoanCompleted(srn, index) -> SectionCompleted
```

Comment:
Needs some change in "transformToEtmp" too in this class to not send partially completed loans to ETMP.  "When transforming data to ETMP only entities where `<entity>Progress` is “JourneyCompleted” are transformed. (in current codebase this might be based on other fields too)"
Otherwise other sections' POST to ETMP will be broken when there is an incomplete loan.

Reply:
This will be addressed

Reply:
Sorry, I wasn't entirely correct in my above comment. It is all true for the regular journey, however not for pre-population. Pre-population should allow partially completed entities to be pushed to ETMP. FYI @BabuRupoll7623